### DISCUSSION TITLE
Github organization

### DISCUSSION THREAD

**Discussion by ughurabbasov** [community/community#135260] [Enterprise]
### Select Topic Area

Show & Tell

### GitHub Feature Area

Discussions

### Body

Guys i have my unique business name. My domain name is unique my business name is unique but in github when i wanted to create organization with my business name some guy in bangladesh took it already. It has no member but have some html activities.  Unlike him i got a real business name and want to put officially my business name in github organizations but i cant because of some guy. Is this issue resolvable?? Because for me it is soo important to put my official business name!

**Comment by queenofcorgis** [comment]
Hey there!  👋 

Thanks for posting in the GitHub Community, @UgurAbasov ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!

**Reply by ughurabbasov** [reply]
Thank you soo much i appreciate it

**Comment by everforyou** [comment]
It sounds frustrating! Unfortunately, GitHub's policy is that organization names are unique, and if someone else has claimed it—even if they have no active members—you can't use that name.

You might consider reaching out to GitHub support to explain your situation. They may be able to help, especially since you have a legitimate business name. Alternatively, you could try a slight variation of your business name for your organization. Good luck!

**Reply by ughurabbasov** [reply]
Thanks, but wont variation of business name affect reputation and validity of my organization? Like for example google doesnt have google123 in their org name or apple doesnt have appleteam. Their name is same as their business name. But if my business name is for example bla and putting in the name bla123 will be a good choice? I dont think so.  And it is soo bad that even if i have my unique name everywhere except github

**Reply by ughurabbasov** [reply]
Lol I checked now the owner and he is under 13 years old, I made report. And GitHub policy says that if user under 13 we will terminate account immediately. If this will happen will organization name lost so that I can take it after 90 days?
